### PR TITLE
initial setup

### PR DESCRIPTION
**Pull request by raniarusly**

### CODE REVIEW

**Review by lornaarmstrong — APPROVED** [review]
LGTM!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target/
```
Comment (MostafaGabriel) [inline .gitignore:1]: why do we need to ignore this?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+    <dependency>
+        <groupId>com.google.appengine</groupId>
+        <artifactId>appengine-api-1.0-sdk</artifactId>
+        <version>1.9.59</version>
```
Comment (MostafaGabriel) [inline pom.xml]:
```suggestion
      <groupId>com.google.appengine</groupId>
      <artifactId>appengine-api-1.0-sdk</artifactId>
      <version>1.9.59</version>
```

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+          <deploy.projectId>student-recommendations</deploy.projectId>
+          <deploy.version>1</deploy.version>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```
Comment (MostafaGabriel) [inline pom.xml]: please add a new line at the end of the file :)

**Review by MostafaGabriel — COMMENTED** [review]
Thanks Rania!

File: .gitignore (hunk 1 above)
Comment (raniarusly) [reply .gitignore:1]: the target folder is the maven default output folder. I think it s normal to ignore maven target, just like on the step portfolio website. I followed it : )

Comment (MostafaGabriel) [reply .gitignore:1]: Got it. Thanks! :D

**Review by MostafaGabriel — APPROVED** [review]

**Pull request merged**